experimental/LieAlgebras/src/WeylGroup.jl

```diff
@@ -1,3 +1,189 @@
+@doc raw"""
+    exchange!(W::WeylGroup, i::UInt8, w::AbstractVector{UInt8}) -> Bool
+
+Return `true`, if `w` could be transformed into an equivalent word with `i` at the first position,
+otherwise `false`.
+
+See also [`exchange!(::WeylGroup, ::Vector{UInt8}, ::UInt8)`](@ref).
+"""
+function exchange!(W::WeylGroup, i::UInt8, w::AbstractVector{UInt8})
```

Comment:
A function returning a bool, usually is named as `is_*` or `can_*` or something similar.